src/server/saved_objects/client/saved_objects_client.js

```diff
@@ -122,7 +123,8 @@ export class SavedObjectsClient {
         body: {
           type,
           updated_at: time,
-          [type]: attributes
+          [type]: attributes,
+          ...extraBodyProperties
```

Comment:
Used to assign the `spaceId`

Reply:
Should we put this first so it can't overwrite the other properties?